test/functional/test_framework/test_framework.py

```diff
@@ -666,6 +671,10 @@ def is_wallet_compiled(self):
 
         return config["components"].getboolean("ENABLE_WALLET")
 
+    def is_wallet_tool_compiled(self):
+        """Checks whether bitcoin-wallet was compiled."""
```

Comment:
17497: dashify

Reply:
done, squashed

Reply:
actually done :see_no_evil: 

I'm just making sure you're actually reviewing :P